packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_account_future_swap.test.ts

```diff
@@ -160,6 +161,14 @@ async function checkTxouts (outs: TxOut[]): Promise<void> {
   expect(prevouts[0].value.toNumber()).toBeGreaterThan(9.999)
 }
 
+// send transaction without minting a single block in the process
+async function sendTransactionWithoutBlockMint (transaction: TransactionSegWit): Promise<void> {
```

Comment:
What if we move this function to test.utils.ts, then it will be common for all.

Reply:
yeah, can add it if other tests need it in the future. But we don't have "within the same block" tests elsewhere I think.